active/0000-remove-ember-cli-eslint.md

```diff
@@ -0,0 +1,40 @@
+- Start Date: 2018-08-13
+- RFC PR: (leave this empty)
+
+# Summary
+
+Remove https://github.com/ember-cli/ember-cli-eslint from projects `ember-cli` generates.
+
+# Motivation
+
+1. Improve our build speed
+2. Hacks required to support features such as [PR #122
+   broccoli-lint-eslint](https://github.com/ember-cli/broccoli-lint-eslint/pull/122#discussion-diff-153937455R28)
+3. Simplicity. `eslint` is common among JS stack, and integrations with editors
+   / precommit-hooks are ubiquitous. Removing this layer of abstraction will
+   simplify how `eslint` is used throughout `ember-cli`.
+
+# Detailed design
+
+1. Change blueprint to pull in `eslint` as opposed to `ember-cli-eslint` under
+   `devDependencies`.
+2. Provide documentation on `eslint` and editor integration as well as precommit hooks
+
```

Comment:
Maybe:
3. Add to the default Travis blueprint:

```
script:
  - yarn lint:js
  - yarn test
```

To make it clear that linting is still running (and still part of the test process) it just doesn't run along side the other tests.

Reply:
I think this is a good idea. Thanks @jrjohnson!

Reply:
@jrjohnson - You are totally correct! So much so, that its **already what we do!**

https://github.com/ember-cli/ember-new-output/blob/v3.4.1/.travis.yml#L25-L27